dm/worker/subtask.go

```diff
@@ -692,3 +693,11 @@ func (st *SubTask) HandleError(ctx context.Context, req *pb.HandleWorkerErrorReq
 	}
 	return err
 }
+
+func updateTaskState(task, sourceID string, stage pb.Stage) {
+	if stage == pb.Stage_Stopped || stage == pb.Stage_Finished {
+		taskState.DeleteAllAboutLabels(prometheus.Labels{"task": task, "source_id": sourceID})
```

Comment:
we might update the description of metics in dm.json and dm_instance.json

"The current state of subtasks..."